libqtile/backend/x11/core.py

```diff
@@ -328,12 +331,7 @@ def _xpoll(self) -> None:
             except Exception:
                 error_code = self.conn.conn.has_error()
                 if error_code:
-                    error_string = xcbq.XCB_CONN_ERRORS[error_code]
-                    logger.exception(
-                        "Shutting down due to X connection error %s (%s)",
-                        error_string,
-                        error_code,
-                    )
+                    logger.warning("Shutting down due to disconnection from X server")
```

Comment:
How do we know it's disconnection-related error?

Reply:
I assumed that was the case based on the log message that I replaced